indico/modules/events/registration/controllers/display.py

```diff
@@ -155,6 +155,8 @@ def _process(self):
             user_data.update((attr, getattr(self.invitation, attr)) for attr in ('first_name', 'last_name', 'email'))
         return self.view_class.render_template('display/regform_display.html', self.event, event=self.event,
                                                sections=get_event_section_data(self.regform), regform=self.regform,
+                                               payment_conditions=event_settings.get(self.event, 'conditions'),
```

Comment:
The payment settings should be imported like this:

    from indico.modules.payment import event_settings as payment_event_settings

Right now it's absolutely not clear which settings you have without looking at the imports (eventually we should rename it in payment/__init__.py and other modules defining settings proxies like this). Not sure what I was thinking when not using proper names while adding those...